### ISSUE TITLE
Nothing happens in my project when opening folder or opening recursively

### ISSUE THREAD

**Issue by lostchopstik** [kieran-osgood/vscode-open-files-in-directory#4]
Nothing too complex about my folders, its a React Native project so I'm trying to open a source folder containing a ton of JS and TSX files.

![2021-09-01_13-26-57](https://user-images.githubusercontent.com/3332512/131740253-e8c3d378-9c73-49df-92b7-d2c14e4d8d21.gif)

**Comment by kieran-osgood** [comment]
Could you try capturing the output in the output tab for the extension so I can try to reproduce/narrow it down, 
I'm not experiencing this issue (there is an issue #3 which seems to be regarding multi-root workspaces which I haven't been able to test yet).

Open the output tab on the `Log (Extension Host)` filter:
![image](https://user-images.githubusercontent.com/17687552/136747287-853e5c68-fbcc-4b9c-afcf-07ed3700f268.png)
Right click in the window and `Clear Output`
![image](https://user-images.githubusercontent.com/17687552/136747388-e548a57e-0cdd-48d6-a19f-a6f94a446c69.png)
Perform one of the open file actions, and copy the output into a snippet here.
 
N.B. I missed the notification for this originally - apologies for the delay

**Comment by ryota-abe** [comment]
same problem.
- prepare a file `C:\test\x\a.txt`
- `File > Open Folder...` and select `C:\test`
- right click the folder `x` in EXPLORER view
- `Open all files in directory`
- nothing happens 🤔 
```
[2021-12-24 09:55:49.095] [exthost] [error] Error: ENOENT: no such file or directory, stat '/c:/test/x'
    at statSync (fs.js:1086:3)
    at Object.e.statSync (electron/js2c/asar_bundle.js:5:4315)
    at openAllFiles (c:\Users\ryota-abe\.vscode\extensions\ko-dev.vscode-open-files-in-directory-1.0.1\out\commands.js:25:28)
    at openCurrentDirectoryFiles (c:\Users\ryota-abe\.vscode\extensions\ko-dev.vscode-open-files-in-directory-1.0.1\out\commands.js:13:5)
    at r._executeContributedCommand (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:95:31080)
    at r.$executeContributedCommand (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:95:31622)
    at t._doInvokeHandler (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:99:13802)
    at t._invokeHandler (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:99:13486)
    at t._receiveRequest (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:99:12148)
    at t._receiveOneMessage (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:99:10826)
    at c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:99:8922
    at u.fire (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:66:1712)
    at r.fire (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:74:18942)
    at c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:114:34341
    at u.fire (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:66:1712)
    at r.fire (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:74:18942)
    at n._receiveMessage (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:74:23523)
    at c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:74:21057
    at u.fire (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:66:1712)
    at w.acceptChunk (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:74:15770)
    at c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:74:14900
    at Socket.T (c:\Users\ryota-abe\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:114:13813)
    at Socket.emit (events.js:315:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at Socket.Readable.push (internal/streams/readable.js:223:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:188:23) vscode-open-files-in-directory.currentDirFiles {"value":"ko-dev.vscode-open-files-in-directory","_lower":"ko-dev.vscode-open-files-in-directory"}
```

**Comment by kieran-osgood** [comment]
Thank you for the context output there it helped me notice you as well as the OP are on windows, guessing it's probably pathing related, will see if I can't patch this then!

**Comment by kieran-osgood** [comment]
Well time got away from me with starting a new job, but I just published an update in v1.0.2 which I hope fixes it, I don't have a windows machine to hand (will have one in 2-3 weeks so will double check then)

**Comment by lostchopstik** [comment]
I tried the latest and was not successful on Windows. It looks like the base is repeated in the path it's attempting to open.
![image](https://user-images.githubusercontent.com/3332512/177397711-e08dfb87-72f5-43d1-a8ba-fdf0fd7a5163.png)

**Comment by kieran-osgood** [comment]
I finally got access to a windows machine, so I believe this should (finally) be resolved in v1.0.4!
